src/settings/index.ts

```diff
@@ -33,6 +33,7 @@ export interface Settings {
         [stopPlaceId: string]: string[]
     }
     distance?: number
+    zoom?: number
```

Comment:
Er det bedre å kalle det `scootersZoom` eller noko sånt dersom me skal ha zoom for andre ting seinare? Om det blir kart-tile for bysykkel f.eks., så er det ikkje nødvendigvis samme zoom-nivå som er ønskelig?